locale/de_DE/LC_MESSAGES/djangojs.po

```diff
@@ -1282,3 +1195,56 @@ msgid "More information can be found in the privacy policy of Vimeo under: "
 msgstr ""
 "Weitere Informationen finden Sie in der Datenschutzerklärung von Vimeo "
 "unter: "
+
```

Comment:
@fuzzylogic2000 this is all the deletd ones :(

Reply:
@phillimorland Yes, and there are even more in another commit. But @sabinammm is on it and will put them all back!

Reply:
@fuzzylogic2000 actually I checked translation PRs through November 2021 and this is the only commit with automatically removed translations that I am re-adding. Is there another one that I missed?

Reply:
@sabinammm In the other commit/PR, I posted on taiga, we did not loose any translations, but only changed all the strings that were added to the old ones? 
commit https://github.com/liqd/a4-meinberlin/commit/4ab744ae8d1b52667964bf3b078fd2c6862ed441#diff-2833e2202830b2667d27de3334567174802d8e78f0ce316657b743fdcf47d038
from https://github.com/liqd/a4-meinberlin/pull/4055

Reply:
@fuzzylogic2000 Oh, thanks for the repost :) The strings that were removed in that [commit](https://github.com/liqd/a4-meinberlin/commit/4ab744ae8d1b52667964bf3b078fd2c6862ed441#diff-2833e2202830b2667d27de3334567174802d8e78f0ce316657b743fdcf47d038), are now coming from a4 and are all there in the current translation files. These are also all in js-files. I only found 1 string that was removed in the other commit that comes from an a4-jsx-file. So I think we are good to go at this time, right?